configs/protocol.json

```diff
@@ -2,6 +2,7 @@
   "ProtocolConfiguration": {
     "Magic": 56753,
     "AddressVersion": 23,
+	"SecondsPerBlock": 5,

```

Comment:
let's keep this as the default of 15 to ensure privatenet stays aligned with coznet/testnet/mainnet

Reply:
Btw, I just tried changing this `SecondsPerBlock` setting, and it does not seem to have any effect. Have you tested it?

Reply:
Not tested the setting myself 🤔

Reply:
This is not used in neo-python but with the official neo-cli consensus nodes. I've tried changing that setting in `protocol.json` to anything between 5 and 60 seconds, but keep seeing 15 seconds block times. Curious, would be very useful!